Cartfile.resolved

```diff
@@ -1,4 +1,4 @@
-github "reitermarkus/DDC.swift" "41e7c49b0450033c5349ca1cf5234a26ebc011b8"
+github "reitermarkus/DDC.swift" "1763870c94c555ff93878caaec8235fd3a9a429d"
```

Comment:
Can you revert this from the commit? I'd prefer it to be just the translation, probably need to use `carthage bootstrap` instead of `carthage update`.